MatomoTracker.php

```diff
@@ -1979,6 +1979,8 @@ protected function sendRequest($url, $method = 'GET', $data = null, $force = fal
 
             $this->parseIncomingCookies(explode("\r\n", $header));
 
+            curl_close($ch);
```

Comment:
you might need to wrap the code above into a `try` and put the closing into a `finally`, otherwise the connection might still remain open if it returns an error, as that would throw an exception before the connection is closed.

Reply:
@sgiehl agree, nice catch)